### PR TITLE
clarify which credential format each example uses

### LINKED ISSUE #316: Example in Credential Issuer Metadata Parameters shows w3c credential
The "non-normative example of Credential Issuer metadata" in [Credential Issuer Metadata Parameters](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-credential-issuer-metadata-p) should state that it shows a [W3C Verifiable Credentials](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-w3c-verifiable-credentials).

Or should this be updated to use the newer [IETF SD-JWT VC](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#name-ietf-sd-jwt-vc) format?

### PR DESCRIPTION
**Pull request by Sakurann**
resolves #316.

- i switched the issuer metadata example to sd-jwt vc as suggested (involved updating issuer metadata for sd-jwt vc a little bit)
- also clarified in all other examples in main text what credential format is used, if it was not clarified.

### CODE REVIEW

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1390,9 +1390,9 @@ Depending on the Credential Format, additional parameters might be present in th
 
 The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.
 
-The following is a non-normative example of Credential Issuer metadata:
+The following is a non-normative example of Credential Issuer metadata with Credential format `vc+sd-jwt`:
```
Comment (tplooker) [inline openid-4-verifiable-credential-issuance-1_0.md:1393]:
```suggestion
The following is a non-normative example of Credential Issuer metadata with the Credential format `vc+sd-jwt`:
```

**Review by tplooker — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -1059,7 +1059,7 @@ The following parameters are used in the Batch Credential Request:
 
 * `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request).
 
-Below is a non-normative example of a Batch Credential Request:
+Below is a non-normative example of a Batch Credential Request for two Credentials in an IETF SD-JWT VC [@VC_DATA] format and [@ISO.18013-5] format:
```
Comment (javereec) [inline openid-4-verifiable-credential-issuance-1_0.md:1062]:
```suggestion
Below is a non-normative example of a Batch Credential Request for two Credentials in an IETF SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc] format and [@ISO.18013-5] format:
```

**Review by c2bo — APPROVED** [review]
Looks good with the proposed changes

**Review by paulbastian — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -1059,7 +1059,7 @@ The following parameters are used in the Batch Credential Request:
 
 * `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request).
 
-Below is a non-normative example of a Batch Credential Request:
+Below is a non-normative example of a Batch Credential Request for two Credentials in an [@VC_DATA] format and [@ISO.18013-5] format:
```
Comment (javereec) [inline openid-4-verifiable-credential-issuance-1_0.md:1062]: I find this sentence hard to read, suggest the following

```suggestion
Below is a non-normative example of a Batch Credential Request for two Credentials; one in [@VC_DATA] format and another in [@ISO.18013-5] format:
```

**Review by javereec — COMMENTED** [review]
Improve readability

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 3 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md]: for the future guidance.. for editorial suggestions, would really appreciate approval conditional to suggested changes to being accepted. if I agree with the changes, I can just accept them and merge a PR without asking to re-review, and if i do not agree, I will reach back to you to clarify until we reach agreement.

**Review by javereec — APPROVED** [review]

**Review by jogu — APPROVED** [review]

**Comment by jogu** [comment]
Unfortunately there's a conflict in this now :(

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 4]
```diff
@@ -1088,10 +1088,7 @@ A Client submits a Batch Credential Request to the Batch Credential Endpoint by
   * `proofs`: OPTIONAL. See (#credential-request).
 * `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.
 
-Below is a non-normative example of a Batch Credential Request requesting:
-
-* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
-* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
+Below is a non-normative example of a Batch Credential Request for two Credentials; one in [@VC_DATA] format and another in [@ISO.18013-5] format:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1091]:
```suggestion
Below is a non-normative example of a Batch Credential Request for four Credentials, all in [@VC_DATA] format:
* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 5]
```diff
@@ -438,7 +438,7 @@ The following is a non-normative example of an `authorization_details` object wi
 
 <{{examples/authorization_details.json}}
 
-The following is a non-normative example of an `authorization_details` object with a `format`:
+The following is a non-normative example of an `authorization_details` object with a `format` parameter of the Credential format `vc+sd-jwt`:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:444]:
```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 6]
```diff
@@ -1443,9 +1440,9 @@ Depending on the Credential Format, additional parameters might be present in th
 
 The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.
 
-The following is a non-normative example of Credential Issuer metadata:
+The following is a non-normative example of Credential Issuer metadata with the Credential format `vc+sd-jwt`:
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1443]:
```suggestion
The following is a non-normative example of Credential Issuer metadata of a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 7]
```diff
@@ -1088,10 +1088,10 @@ A Client submits a Batch Credential Request to the Batch Credential Endpoint by
   * `proofs`: OPTIONAL. See (#credential-request).
 * `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.
 
-Below is a non-normative example of a Batch Credential Request requesting:
+Below is a non-normative example of a Batch Credential Request requesting four Credentials:
 
-* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
-* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
+* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
+* Two Credentials for the same Credential Configuration but with different Credential Dataset in [@VC_DATA] format.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1094]: @paulbastian can you please re-review if this new text makes sense

**Comment by Sakurann** [comment]
sorry, I had to change the examples based on the paul's PR, please re-review..

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 7 above)
Comment (paulbastian) [reply openid-4-verifiable-credential-issuance-1_0.md:1094]: Makes sense

**Review by paulbastian — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 8]
```diff
@@ -441,7 +441,7 @@ The following is a non-normative example of an `authorization_details` object wi
 
 <{{examples/authorization_details.json}}
 
-The following is a non-normative example of an `authorization_details` object with a `format`:
+The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```
Comment (tplooker) [inline openid-4-verifiable-credential-issuance-1_0.md:444]:
```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 9]
```diff
@@ -1088,10 +1088,10 @@ A Client submits a Batch Credential Request to the Batch Credential Endpoint by
   * `proofs`: OPTIONAL. See (#credential-request).
 * `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.
 
-Below is a non-normative example of a Batch Credential Request requesting:
+Below is a non-normative example of a Batch Credential Request requesting four Credentials:
 
-* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
-* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
+* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
```
Comment (tplooker) [inline openid-4-verifiable-credential-issuance-1_0.md:1093]:
```suggestion
* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 7 above)
Comment (tplooker) [inline openid-4-verifiable-credential-issuance-1_0.md:1094]:
```suggestion
* Two Credentials for the same Credential Configuration but with different Credential Datasets in the [@VC_DATA] format.
```

**Review by tplooker — APPROVED** [review]
Sorry I know its nit picky but just correcting the usage of articles here, with these adjustments I'm happy to approve

**Comment by Sakurann** [comment]
@tplooker thanks a lot for nitpicking! appreciate it!

File: examples/credential_metadata_sd_jwt_vc.json [hunk 10]
```diff
@@ -12,11 +12,22 @@
       "display": [
         {
           "name": "IdentityCredential",
+          "logo": {
+                    "uri": "https://university.example.edu/public/logo.png",
+                    "alt_text": "a square logo of a university"
+                  },
```
Comment (jogu) [inline examples/credential_metadata_sd_jwt_vc.json:18]: Trivial fix of indentation:

```suggestion
          },
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 8 above)
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:444]: I think this reads more naturally without the second 'an' (hopefully I'm not contradicting other people's suggestions here...):

```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 9 above)
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1093]: Same:

```suggestion
* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 11]
```diff
@@ -1443,9 +1438,9 @@ Depending on the Credential Format, additional parameters might be present in th
 
 The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.
 
-The following is a non-normative example of Credential Issuer metadata:
+The following is a non-normative example of Credential Issuer metadata of a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1441]:
```suggestion
The following is a non-normative example of Credential Issuer metadata of a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

**Review by jogu — APPROVED** [review]
Looks good to me, made a couple of very small suggestions.

**Pull request merged**